security/src/main/java/io/micronaut/security/endpoints/introspection/IntrospectionController.java

```diff
@@ -22,26 +22,23 @@
 import io.micronaut.http.HttpResponse;
 import io.micronaut.http.MediaType;
 import io.micronaut.http.MutableHttpResponse;
-import io.micronaut.http.annotation.Body;
-import io.micronaut.http.annotation.Consumes;
-import io.micronaut.http.annotation.Controller;
-import io.micronaut.http.annotation.Get;
-import io.micronaut.http.annotation.Post;
+import io.micronaut.http.annotation.*;
```

Comment:
is the wildcard import intentional?

Reply:
It was the IDE optimising them. Wildcard imports are allowed as per our Checkstyle config.